lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -524,6 +527,47 @@ def test_submission_history_xss(self):
         response = self.client.get(url)
         self.assertFalse('<script>' in response.content)
 
+    def test_submission_history_contents(self):
+        # log into a staff account
+        admin = AdminFactory.create()
+
+        self.client.login(username=admin.username, password='test')
+
+        usage_key = self.course_key.make_usage_key('problem', 'test-history')
+        state_client = DjangoXBlockUserStateClient(admin)
+
+        # store state via the UserStateClient
+        state_client.set(
+            username=admin.username,
+            block_key=usage_key,
+            state={'field_a': 'x', 'field_b': 'y'}
```

Comment:
I like this naming convention for the dicts more than the earlier versions that were things like {'a':1, 'b';2}. This is easier to read.